client/analytics/components/report-summary/index.js

```diff
@@ -70,10 +73,27 @@ export class ReportSummary extends Component {
 }
 
 ReportSummary.propTypes = {
+	/**
+	 * Properties of all the charts available for that report.
+	 */
 	charts: PropTypes.array.isRequired,
+	/**
+	 * The endpoint to use in API calls.
```

Comment:
I wonder if more description would be useful here. What does "endpoint" mean in this context. Maybe provide an example?

Reply:
Done! :slightly_smiling_face: 

a1df848a2b944c1985e5ac96869b1abe2d2662fb